File: instructor/06_colecciones/04_diccionario.py
provincias={'02':'Pichincha',
            '04':'Guayas',
            '06':'Carchi'
            }

print('---------------- DICCIONARIO ----------------')

print('Crear Provincias -> ',provincias)

print('Primera Provincia -> ',provincias.get('02'))

provincias['02']='PICHINCHA'
print('Cambio Primera Provincia -> ',provincias.get('02'))

provincias['08']='Azuay'
provincias['08']='Azuay'
provincias['0802']='Azuay'
print('Nueva Provincia -> ',provincias)

print('Numero Provincias',len(provincias))

del(provincias['04'])
print('Quitar Provincia')
print('Numero Provincias',len(provincias))
print('Provincias Final -> ',provincias)

print('Recorrer Provincias')
for codigo,provincia in provincias.items() :
    print('Provincia =', codigo,provincia)

File: instructor/99_js/06_colecciones/04_diccionario.js
let provincias=new Map([
            ['02','Pichincha'],
            ['04','Guayas'],
            ['06','Carchi']
            ])

console.log('---------------- DICCIONARIO ----------------')

console.log('Crear Provincias -> ',provincias)

console.log('Primera Provincia -> ',provincias.get('02'))

provincias.set('02','PICHINCHA')
console.log('Cambio Primera Provincia -> ',provincias.get('02'))

provincias.set('08','Azuay')
provincias.set('08','Azuay')
console.log('Nueva Provincia -> ',provincias)

console.log('Numero Provincias',provincias.size)

delete provincias.delete('04')
console.log('Quitar Provincia')
console.log('Numero Provincias',provincias.size)
console.log('Provincias Final -> ',provincias)

console.log('Recorrer Provincias')
for(let [codigo,provincia] of provincias.entries()) 
    console.log('Provincia =', codigo,provincia)